tests/test_hf_api.py

```diff
@@ -140,18 +140,12 @@ def test_repo_id_no_warning():
     # tests that passing repo_id as positional arg doesn't raise any warnings
     # for {create, delete}_repo and update_repo_visibility
     api = HfApi(endpoint=ENDPOINT_STAGING, token=TOKEN)
-    REPO_NAME = repo_name("crud")
 
-    args = [
-        ("create_repo", {}),
-        ("update_repo_visibility", {"private": False}),
-        ("delete_repo", {}),
-    ]
-
-    for method, kwargs in args:
-        with warnings.catch_warnings(record=True) as record:
-            getattr(api, method)(REPO_NAME, repo_type=REPO_TYPE_MODEL, **kwargs)
-        assert not len(record)
+    with warnings.catch_warnings(record=True) as record:
+        repo_id = api.create_repo(repo_name()).repo_id
+        api.update_repo_visibility(repo_id, private=True)
+        api.delete_repo(repo_id)
+    assert not len(record)
```

Comment:
much more readable :)

Reply:
ahah yes, I had to figure out myself what the test was doing initially :smile:  (2 years old so... ^^)